.gitignore

```diff
@@ -65,4 +66,4 @@ target/
 .ipynb_checkpoints
 
 # Mac meta
-.DS_Store
+.DS_Store
```

Comment:
This removal of the trailing new line character is unnecessary. POSIX standard requires even the last line of a file to have a trailing new line character.

Reply:
Agreed, I think this is an artifact of my PyCharm or something. I am adding the newline with a new commit :)